### PR TITLE
Explain counter and distribution

### PR DESCRIPTION
**Pull request by lei-tang**
Please provide a description for what this PR is for.
Fix https://github.com/istio/istio.io/issues/9027

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/reference/config/metrics/index.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ aliases:
 The following are the standard service level metrics exported by Istio.
 
 The telemetry component is [implemented](https://github.com/istio/proxy/tree/master/extensions/stats) as a [Proxy-wasm](https://github.com/proxy-wasm/spec) plugin.
+A `COUNTER` is a strictly increasing integer.
```
Comment (zirain) [inline content/en/docs/reference/config/metrics/index.md:15]: what about reference to prometheus/OpenTracing or OpenTelemetry docs?

Comment (lei-tang) [reply content/en/docs/reference/config/metrics/index.md:15]: Based on the code, `COUNTER` and `DISTRIBUTION` correspond to the metrics counter and histogram
in [the Envoy document](https://github.com/envoyproxy/envoy/blob/main/source/docs/stats.md). Therefore, the Envoy document is referred here.

**Review by zirain — APPROVED** [review]
lgtm

**Comment by lei-tang** [comment]
@istio/wg-docs-maintainers-english Can you take a look?

**Pull request merged**